app/frontend/src/features/messages/groupchats/CreateGroupChat.test.tsx

```diff
@@ -0,0 +1,38 @@
+import { render, screen } from "@testing-library/react";
+import { FRIENDS, NEW_CHAT } from "features/messages/constants";
+import CreateGroupChat from "features/messages/groupchats/CreateGroupChat";
+import { User } from "pb/api_pb";
+import users from "test/fixtures/users.json";
+import { getHookWrapperWithClient } from "test/hookWrapper";
+
+describe("CreateGroupChat with router state", () => {
+  beforeEach(() => {
+    const { wrapper } = getHookWrapperWithClient({
+      initialRouterEntries: [
+        {
+          pathname: "create-group-chat-test",
+          state: { createMessageTo: users[0] as User.AsObject },
+        },
+      ],
+    });
+    render(<CreateGroupChat />, { wrapper });
+  });
+
+  it("initially shows the create dialog with a user pre-filled", async () => {
+    expect(screen.getByLabelText(FRIENDS)).toBeVisible();
+    expect(screen.getByText(users[0].name)).toBeVisible();
+    expect(screen.getByText(NEW_CHAT, { selector: "h2" })).toBeVisible();
```

Comment:
```suggestion
    expect(screen.getByRole('heading', { level: 2, name: NEW_CHAT })).toBeVisible();
```
Better to use `getByRole` here so it checks that it's accessible through the accessibility tree by things like screen readers.